lib/libc/stdlib/malloc/snmalloc/Makefile.inc

```diff
@@ -0,0 +1,45 @@
+.PATH: ${LIBC_SRCTOP}/stdlib/malloc/snmalloc
+
+MISRCS+=	malloc.cc
+
+# Specify the locations in contrib for headers.
+CXXFLAGS+=	-I${SRCTOP}/contrib/subrepo-snmalloc/src/snmalloc
```

Comment:
How come this wasn't scoped?

Reply:
I think I was thinking of it as like .PATH and thus global to the libc build.